common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -1156,6 +1157,7 @@ def _update_module_location(module, new_location):
             module.get_explicitly_set_fields_by_scope(Scope.content).keys() +
             module.get_explicitly_set_fields_by_scope(Scope.settings).keys()
         )
+        original_module = copy.deepcopy(module)
```

Comment:
is it possible to get the data we want out of the module without copying it? Or is this the right way to do this? @doctoryes